proto/internalapi/central/secured_cluster_cert_refresh.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package central;
+
+import "storage/service_identity.proto";
+
+option go_package = "./internalapi/central;central";
+
+message SecuredClusterCertsIssueError {
+  string message = 1;
+}
+
+message IssueSecuredClusterCertsRequest {
```

Comment:
I think this service could be generalized for all certificates by adding a service type parameter to the request:

```
enum ServiceType {
    CollectorService =1;
    SensorService = 2;
    ScannerService = 3; 
    // ...
}

message IssueSecuredClusterCertsRequest {
    string request_id = 1;
    ServiceType service_type = 2;
}
```

Reply:
For simplicity I wanted to follow the existing approach, which is a bit more high level.

The already existing `IssueLocalScannerCertsRequest` returns all certificates required for local scanner (scanner v2 + optionally scanner v4 if enabled)

So I wanted `IssueSecuredClusterCertsRequest` to do the same for Sensor (return all the certs needed). In the future I would even include the scanner certs here, and deprecate the other call.

Reply:
I should also add that we already have logic on Sensor side that retrieves all the (localscanner) certs in a single gRPC call and saves them into secrets, and my plan is to reuse that. Making the API more granular would add more implementation effort on the Sensor side (and I'm not sure of the benefits, I think it's fine to refresh all certs together).

Reply:
I really do like this approach! It's really simple and re-uses the existing code and returns all certs in one request.